### PR TITLE
Build release wheels

### PR DESCRIPTION
**Pull request by rcoh**

### CODE REVIEW

File: semgrep/setup.py [hunk 1]
```diff
@@ -1,15 +1,118 @@
...
 
-setuptools.setup(
+setup(
     name="semgrep",  # Replace with your own username
     version="0.6.0",
     author="Russell & Return 2 Corp",
-    author_email="author@example.com",
-    description="semgrep python wrapper",
-    long_description="bloop",
+    author_email="support@returntocorp.com",
```
Comment (brendongo) [inline semgrep/setup.py]:
```suggestion
    author_email="support@r2c.dev",
```

File: semgrep/requirements.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ pytest==5.3.5
 pyyaml==5.3
 requests==2.22.0
 tox==3.14.6
+wheel
```
Comment (brendongo) [inline semgrep/requirements.txt]: No version pinning?

File: .bento/config.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ tools:
     - subprocess-without-shell-equals-true
     - try-except-continue
     - urllib-urlopen
-    run: true
+    run: false
```
Comment (brendongo) [inline .bento/config.yml]: Can we leave this on and archive the relevant places this fires on this PR

Comment (rcoh) [reply .bento/config.yml]: done

**Review by brendongo — APPROVED** [review]

**Pull request merged**